docs/EN_US/ECLWatch/TheECLWatchMan.xml

```diff
@@ -685,6 +685,15 @@
       page in ECL Watch click on the <emphasis role="bold">ECL
       Watch</emphasis> image at the top of any page, as shown above.</para>
 
+      <sect2>
+        <title>Cluster Status Information Graphic</title>
+
+        <para>The Cluster Activity page displays an informative graphic bar
+        along the top where some graphs provide a glimpse of usage activity on
```

Comment:
The Cluster Activity page displays an informative graphic bar along the top where some graphs provide a glimpse of usage activity on the clusters.
s/b
The Cluster Activity page displays graphs along the top showing cluster disk usage.